src/main/java/com/iexec/worker/docker/DockerService.java

```diff
@@ -149,7 +149,7 @@ public DockerRunResponse run(DockerRunRequest dockerRunRequest) {
                 || dockerRunRequest.getMaxExecutionTime() != 0) {
             removeFromRunningContainersRecord(containerName);
         }
-        if (shouldPrintDeveloperLogs(dockerRunRequest)) {
+        if (shouldPrintDeveloperLogs()) {
             String chainTaskId = dockerRunRequest.getChainTaskId();
             if (StringUtils.isEmpty(chainTaskId)) {
                 log.error("Cannot print developer logs [chainTaskId:{}]", chainTaskId);
```

Comment:
Minor:
If you have the time you can add a test which captures the output and verifies that some logs are present when option is enabled (Did quite the same here: https://github.com/iExecBlockchainComputing/iexec-sms/blob/a8fd668b044a50bbc7aa93a55590ccd13d6acf0a/src/test/java/com/iexec/sms/tee/session/gramine/GramineSessionHandlerServiceTests.java#L76)

Reply:
done in ffb55f7